packages/@office-iss/react-native-win32/package.json

```
@@ -95,11 +95,11 @@
     "react-native": "0.74.0-rc.9"
   },
   "beachball": {
-    "defaultNpmTag": "preview",
+    "defaultNpmTag": "latest",
     "disallowedChangeTypes": [
       "major",
       "minor",
-      "patch"
+      "prerelease"
```

I think we want to add these to all the packages too (for future proofing)?

```suggestion
      "prerelease",
      "premajor",
      "preminor",
      "prepatch"
```